### DISCUSSION TITLE
Segmentation with Scribbles on OHIF-viewer

### DISCUSSION THREAD

**Discussion by donalbretti** [Project-MONAI/MONAILabel#1479] [Q&A]
Hey you all!

I want to annotate my own data set (segmentation of eye surface), and want to use the Scribbles Tool (Histogram+GraphCut) with OHIF-viewer. Within this project i have a few questions/issues:

1. How can i choose a label before i start scribbling? The labels i defined in the configs file only appear after i ran the model. 

2. How can i refine a label i just created? Everytime i try to use Scribbles a second time, my labels get duplicated and i have all of my labels twice.

3. Is there a way to adjust the thickness of the brush?

To start the server i use: 
monailabel start_server --app apps/radiology --studies http://127.0.0.1:8042/dicom-web --conf models segmentation

If you can suggest a better way to annotate the data set with OHIF-viewer, that would also be helpful.
Thanks for all answers in advance!

**Comment by diazandr3s** [comment]
Hi @donalbretti,

Have you managed to use scribbles in OHIF viewer? You may find this video useful: https://www.youtube.com/watch?v=Wxmo7MVc7hI&t=1s

Hope this helps,
